sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/TelemetryEventData.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using OpenTelemetry.Logs;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Models
+{
+    internal partial class TelemetryEventData
+    {
+        public TelemetryEventData(int version, LogRecord logRecord) : base(version)
+        {
+            Properties = new ChangeTrackingDictionary<string, string>();
+            Measurements = new ChangeTrackingDictionary<string, double>();
```

Comment:
a separate issue: Given there is no way user can populate Measurements, any way we can avoid newing up this Dictionary? I am assuming we are doing it for each and every telemetry only to just throw it away.

Reply:
+1 to this, but would need to confirm that change doesn't break the generated serialization code. ie: if generated code expects a not-null value.

Reply:
IIRC, null value causes an issue with serialization. That is why we have this in all telemetry types.

Reply:
maybe create a dictionary once, and re-use throughout to avoid null issues. and save on the large number of unwanted heap allocations.